docs/tutorials/add-a-pallet/configure-a-pallet.md

```diff
@@ -103,8 +103,8 @@ impl pallet_balances::Config for Runtime {
     // The FRAME runtime system is used to track the accounts that hold balances.
     type AccountStore = System;
 
-    // No weight information is supplied to the Balances pallet by the Node Template's runtime.
-    type WeightInfo = ();
+    // Weight information is supplied to the Balances pallet by the Node Template's runtime.
+    type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
```

Comment:
I _think_ we actually intend to leave this weightless, as it's intended for testing only, although I would be for moving to using weights... @jimmychu0807 what do you think? 

@narulasrinivas - have you completed thew entire tutorial and can confirm this works as expected?